src/interactive-window/debugger/jupyter/kernelDebugAdapter.ts

```diff
@@ -67,15 +68,19 @@ export class KernelDebugAdapter extends KernelDebugAdapterBase {
         }
     }
 
-    override handleMessage(message: DebugProtocol.ProtocolMessage): Promise<KernelMessage.IDebugReplyMsg | undefined> {
+    override handleMessage(message: DebugProtocol.ProtocolMessage) {
```

Comment:
Just curious why you don't like method return signatures, I always add them, just my personal preference

Reply:
i think we should change this, as is not supposed to be async.
ie i think we should change this to sync, probably should have left it there's as the code is still async.